packages/php/email-editor/.wp-env.json

```diff
@@ -13,12 +13,10 @@
 	},
 	"env": {
 		"development": {
-			"port": 8887,
-			"phpmyadminPort": 9070
```

Comment:
In case you folks need it, for development env you can still keep using phpMyAdmin - in CI we start only test containers without it, but locally under `development` it should keep working.